circle.yml

```diff
@@ -208,6 +208,7 @@ jobs:
     environment:
       BUILD_TYPE: RelWithDebInfo
       CMAKE_OPTIONS: -DENABLE_ASSERTIONS=ON -DSANITIZE=address,undefined,implicit-conversion,nullability
+      UBSAN_OPTIONS: halt_on_error=1
```

Comment:
Will this also be triggered for the `spectest` job?

Reply:
Sure.

Reply:
Can you remove the commented out stuff from spectest job then?